src/styles/_catalog.scss

```diff
@@ -13,10 +12,17 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width: 489px) {
+    grid-template-columns: repeat(2, auto);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, auto);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, auto);
```

Comment:
The grid-template-columns property should be set to 200px instead of auto. This is to ensure that the card width is always 200px including borders as specified in the task description.